cmd/defaults.go

```diff
@@ -186,6 +186,7 @@ func LoadOrInit(configDir string, dataDir string, keystoreDir string) (*ToolConf
 		return nil, err
 	}
 
+	dataDirFlagSet := dataDir != ""
 	if dataDir == "" {
 		dataDir = home.NscDataHome(home.StoresSubDirName)
```

Comment:
wondering if this defaulting should be done here:
https://github.com/nats-io/nsc/blob/main/cmd/root.go#L169

Reply:
(and whether all the tests etc will work with this change)
ie. should be possible to change it and all works correctly

Reply:
Leaving as is because `LoadOrInit` calls `config.load()` which overwrites the `config` variable with the contents from the config file.

This change sets the value of `config.StoreRoot` after reading the config.